llvm/include/llvm/ADT/MapVector.h

```diff
@@ -57,6 +57,8 @@ class MapVector {
     return std::move(Vector);
   }
 
+  ArrayRef<value_type> getArrayRef() const { return Vector; }
```

Comment:
Probably put this into a separate prep patch with a test in llvm/unittests/ADT/MapVectorTest.cpp.